### PR TITLE
Tools for websocket and rabbitmq client testing

### PR DESCRIPTION
**Pull request by ssd04**

### CODE REVIEW

File: tools/wsClient/main.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+			},
+		},
+	}
+
+	ws.SendSubscribeMessage(subscribeEvent)
+
+	for {
+		m, err := ws.ReadMessage()
+		if err != nil {
+			fmt.Println(err.Error())
```
Comment (sstanculeanu) [inline tools/wsClient/main.go:47]: continue?

File: tools/wsClient/main.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+		}
+
+		var reply data.WSEvent
+		err = json.Unmarshal(m, &reply)
```
Comment (sstanculeanu) [inline tools/wsClient/main.go:51]: check error and continue if != nil?

File: tools/wsClient/main.go [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+// SendSubscribeMessage will send subscribe message
+func (ws *wsClient) SendSubscribeMessage(subscribeEvent *data.SubscribeEvent) error {
+	m, err := json.Marshal(subscribeEvent)
+	if err != nil {
+		return err
+	}
+
+	if err := ws.wsConn.WriteMessage(websocket.BinaryMessage, m); err != nil {
```
Comment (sstanculeanu) [inline tools/wsClient/main.go:110]: L110-L114 can be reduced to `return ws.wsConn.WriteMessage(websocket.BinaryMessage, m)`

also, shouldn't mutex lock should be used for this as well?

File: tools/wsClient/main.go [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+	if err != nil {
+		return nil, err
+	}
+
+	return m, nil
+}
+
+// Close will close connection
+func (ws *wsClient) Close() {
+	ws.wsConn.Close()
```
Comment (sstanculeanu) [inline tools/wsClient/main.go:132]: use mutex for this operation too?

File: tools/wsClient/main.go (hunk 2 above)
Comment (ssd04) [reply tools/wsClient/main.go:52]: done

File: tools/wsClient/main.go (hunk 3 above)
Comment (ssd04) [reply tools/wsClient/main.go]: changed

File: tools/wsClient/main.go (hunk 4 above)
Comment (ssd04) [reply tools/wsClient/main.go:139]: done

**Review by sstanculeanu — APPROVED** [review]

**Review by dragos-rebegea — APPROVED** [review]

**Pull request merged**